R/extract.R

```diff
@@ -817,11 +973,30 @@ extract.clmm <- function(model, include.thresholds = TRUE,
   return(tr)
 }
 
+#' \code{\link{extract}} method for \code{clmm} objects
+#'
+#' \code{\link{extract}} method for \code{clmm} objects. These objects are
+#' created by the \code{\link[ordinal]{clmm}} function in the \pkg{ordinal} package.
+#'
+#' @param include.groups Report groups in the GOF block?
+#' @param include.variance Report variance in the GOF block?
+#' @inheritParams extract,lm-method
+#' @inheritParams extract,glm-method
+#' @inheritParams extract,clm-method
+#' @return A \linkS4class{texreg} object.
+#'
+#' @method extract clmm
+#' @aliases extract.clmm
+#' @family extract
+#' @seealso \link{extract}
+#' @author Philip Leifeld
+#'
```

Comment:
Since there are several methods related to similar objects defined in the same package (`ordinal`), it might make sense to document them all on the same help page. Can you look up how I documented the `lme4` methods and do it in a similar way?

Reply:
I'm happy to do it, but I'm not understanding one thing. In `lme4` you start defining the roxygen with:

```
#' \code{\link{extract}} method for \code{merMod} objects
#'
#' \code{\link{extract}} method for \code{merMod} objects, in particular:
#' \itemize{
#'   \item \code{lmerMod} objects created by the \code{\link[lme4]{lmer}}
#'     function in the \pkg{lme4} package
#'   \item \code{glmerMod} objects created by the \code{\link[lme4]{glmer}}
#'     function in the \pkg{lme4} package
#'   \item \code{nlmerMod} objects created by the \code{\link[lme4]{nlmer}}
#'     function in the \pkg{lme4} package
#' }
```
do these functions have `merMod` as a second class after the specific class that each of them has?

There is nothing comparable in the `ordinal` package, hence I'm not sure what to put there. 

Also, I checked the `ordinal` package and there is no function called `sclm`. There are only `clm` and `clmm`. Maybe `sclm` was in an older version? I looked it up to see if by any chance it belongs to another package, but I found nothing. 

Sorry, but I need help on this one! Thanks!!

Reply:
You could put something like `\code{\link{extract}} methods for statistical models in the \pkg{survival} package` in the first line. Then in the next paragraph, you could describe which models and functions you are documenting and link to them. Like I did there. You add this stuff for the first of the three or so extract methods, and the other two don't need this level of detail because you tell Roxygen to document them on the same page as this one.